Client/trunk/ParaEngineClient/BlockEngine/BlockChunk.cpp

```diff
@@ -694,30 +694,39 @@ namespace ParaEngine
 
 	std::vector<Uint16x3> BlockChunk::refreshBlockVisible(uint16_t blockTemplateId)
```

Comment:
I think the new logic is too slow.  Please discuss with me about your algorithm.  Or if you can have some algorithm with the same level of complexity, then I will gladly merge.